src/slave/containerizer/mesos/isolators/cgroups/subsystems/blkio.cpp

```diff
@@ -107,226 +107,205 @@ Future<ResourceStatistics> BlkioSubsystemProcess::usage(
   CgroupInfo::Blkio::CFQ::Statistics totalCfqRecursive;
   CgroupInfo::Blkio::Throttling::Statistics totalThrottling;
 
-  // Get CFQ statistics.
+  // CFQ was removed from kernel 5.x, so be ready to handle missing statistics.
+
   Try<vector<cgroups::blkio::Value>> time = cfq::time(hierarchy, cgroup);
```

Comment:
Is there a way to detect whether CFQ is support or not? If it is not supported, then maybe we can skip calling the following `cfq::` methods.

Reply:
Hm, not really.
The only way would be maybe to look at all block devices `/sys/block/*/queue/scheduler` and see if any of those files contains `cfq`, but I think it's more fragile.

Reply:
OK to merge?

Reply:
If we know the specific kernel version where CFQ was removed, can we use `kernelVersion()` to detect it like [this code](https://github.com/apache/mesos/blob/96339efb53f7cdf1126ead7755d2b83b435e3263/src/slave/containerizer/mesos/isolators/linux/nnp.cpp#L38)?

Reply:
@cf-natali Any comments?

Reply:
Sorry for the delay, I've been busy lately.

It was removed from kernel 5.0: https://github.com/torvalds/linux/commit/f382fb0bcef4c37dc049e9f6963e3baf204d815c

I've updated the change to explicitly change the kernel version - gitlab renders the diff in a sub-optimal way, but the change is actually very small (adding the version check).

Reply:
Thanks for the update.

Do we need to update any tests accordingly?

Reply:
No, the test that was breaking are now passing.